lib/index.js

```diff
@@ -68,6 +68,28 @@ export default function runContentfulExport (usageParams) {
     opts.errorLogFile = opts.exportDir + '/contentful-export-' + Date.now() + '.log'
   }
 
+  if (typeof opts.proxy === 'string') {
```

Comment:
the main entry point is getting spammed by the data validation, mind moving all of the validation in a separate module called `validator'?